build/platform-id.mk

```diff
@@ -313,6 +317,25 @@ ARM_CPU=cortex-m4
 PLATFORM_THREADING=1
 endif
 
+ifeq ("$(PLATFORM_ID)","26")
+PLATFORM=tracker
+PLATFORM_NAME=tracker
+PLATFORM_GEN=3
+PLATFORM_MCU=nRF52840
+PLATFORM_NET=QUECTEL
+PLATFORM_WIZNET=W5500
+STM32_DEVICE=nRF52840
+PRODUCT_DESC=Production Asset Tracker
+USBD_VID_SPARK=0x2B04
+USBD_PID_DFU=0xD01A
+USBD_PID_CDC=0xC01A
+DEFAULT_PRODUCT_ID=$(PLATFORM_ID)
+PLATFORM_DYNALIB_MODULES=$(PLATFORM_NAME)
+ARM_CPU=cortex-m4
+PLATFORM_THREADING=1
```

Comment:
Permanent `SYSTEM_THREAD(ENABLED);`?

Reply:
No, this just means that the platform supports threading. This is defined for all the other Gen2 and Gen3 platforms as well.